zellij-server/src/unit/screen_tests.rs

```diff
@@ -77,6 +77,9 @@ impl ServerOsApi for FakeInputOutput {
     fn get_cwd(&self, _pid: Pid) -> Option<PathBuf> {
         unimplemented!()
     }
+    fn write_to_file(&self, _: std::string::String, _: std::option::Option<std::string::String>) {
+        unimplemented!()
```

Comment:
Why the explicit package path? Can't we just do `String` and `Option<String>`?

Reply:
My tool (`rls`) automatically generated the method. :) I can remove those with the next commit.